Xamarin.Forms.Core/ToastArguments.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms.Internals
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why do you hide this?

Reply:
To be consistent. We use the same attribute for AlertArguments